packages/flutter/lib/src/gestures/hit_test.dart

```diff
@@ -16,11 +16,16 @@ export 'events.dart' show PointerEvent;
 
 /// An object that can hit-test pointers.
 abstract interface class HitTestable {
-  /// Check whether the given position hits this object.
-  ///
-  /// If this given position hits this object, consider adding a [HitTestEntry]
-  /// to the given hit test result.
+  /// Deprecated. Use [hitTestInView] instead.
+  @Deprecated(
+    'Use hitTestInView and specify the view to hit test. '
```

Comment:
Shall we add something like this: To keep the current behavior, use 0 for viewId for the first view.

Reply:
Also to other copies of this method.

Reply:
Done.